<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RCMainWindow</class>
 <widget class="QMainWindow" name="RCMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RCMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0">
    <property name="margin">
     <number>6</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="4" column="0">
     <widget class="QGroupBox" name="b_output">
      <property name="title">
       <string>Ouput</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QTextBrowser" name="t_output">
         <property name="font">
          <font>
           <family>Monospace</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="b_settings">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Run Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Next run number:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="b_runNumber">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>50000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Header text:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTextBrowser" name="t_header">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Monospace</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monospace'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.Lucida Grande UI'; font-size:13pt;&quot;&gt;Default header text.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Plane positions in increasing Z /mm (for file header. Use -1 for not present):</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QWidget" name="widget" native="true">
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QSpinBox" name="b_z0">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="b_z1">
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="b_z2">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="b_z3">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>300</number>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QSpinBox" name="b_z4">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>400</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="b_z6">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>600</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="b_z5">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="b_z7">
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>700</number>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="b_z8">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QSpinBox" name="b_z9">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="4">
           <widget class="QCheckBox" name="b_includeZposns">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Include in file headers (~20/128 characters extra)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="b_forceSaveRecords">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="l_logo">
      <property name="text">
       <string>Timepix3 Telescope Run Control</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="b_actions">
      <property name="title">
       <string>Telescope Commands</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="b_configure">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Configure</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="b_init">
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Initialise</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="b_startRun">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Start Run</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="b_endRun">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Stop Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Notes: the DAQ processes should be started prior to initialising the Run Control. The run number will auto increment. The Z positions should be set if changed.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QGroupBox" name="b_monitoring">
      <property name="title">
       <string>Monitoring</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="b_startMonitoring">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Start Monitoring</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QPushButton" name="b_endMonitoring">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>End Monitoring</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="6">
        <widget class="QPushButton" name="b_quit">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="b_DQM">
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DQM</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="b_FQM">
         <property name="text">
          <string>FQM</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
